egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -593,7 +593,7 @@ def train(args, run_opts):
             chain_lib.combine_models(
                 dir=args.dir, num_iters=num_iters,
                 models_to_combine=models_to_combine,
-                num_chunk_per_minibatch_str=args.num_chunk_per_minibatch,
+                num_chunk_per_minibatch_str="1:64",
```

Comment:
the right fix IMO would be to change this from the calling script by setting the option.  This change causes it to ignore the option, and I'm concerned it would affect existing scripts where the option is used.